india_forecast_app/app.py

```diff
@@ -77,7 +77,7 @@ def get_generation_data(
     if client == "ruvnl":
         start = timestamp - dt.timedelta(hours=1)
     elif client == "ad":
-        start = timestamp - dt.timedelta(hours=3)
+        start = timestamp - dt.timedelta(hours=25)
```

Comment:
Is this change because of data quality issues or we are using a lot more Wind history?

Reply:
Had a look at https://huggingface.co/openclimatefix/windnet_ad_sites/blob/main/data_config.yaml and have answered my own question, makes sense since we have a day of wind history

Reply:
We are using a bunch f wind history yeah, over a day